consensus/consensus.go

```diff
@@ -113,6 +113,12 @@ type Engine interface {
 	// consensus rules that happen at finalization (e.g. block rewards).
 	FinalizeAndAssemble(chain ChainHeaderReader, header *types.Header, state *state.StateDB, txs []*types.Transaction, uncles []*types.Header, etxs []*types.Transaction, manifest types.BlockManifest, receipts []*types.Receipt) (*types.Block, error)
 
+    // Hashrate returns the measured rate of the search invocations
+    // per second over the last minute.
+    // Note the returned hashrate includes local hashrate, but also includes the total
+    // hashrate of all remote miner.
```

Comment:
Nittiest of picks
"remote miners." 
And actually we don't even have remote miners, so can remove this comment entirely.